Stacks/Parenthesis_Checker.cpp

```diff
@@ -0,0 +1,93 @@
+ // Anisha Singh
```

Comment:
Remove your initials in the beginning. Instead add the description of your algorithm using multi-line comments at the starting of the code.
```suggestion
 // Anisha Singh
```

Reply:
I resolved the issues. Thank you for guiding.